oshi-core/src/main/java/oshi/util/Util.java

```diff
@@ -33,73 +37,137 @@
  */
 public class Util {
-    private static final Logger LOG = LoggerFactory.getLogger(Util.class);
-
-    private Util() {
-    }
-
-    /**
-     * Sleeps for the specified number of milliseconds.
-     *
-     * @param ms
-     *            How long to sleep
-     */
-    public static void sleep(long ms) {
-        try {
-            LOG.trace("Sleeping for {} ms", ms);
-            Thread.sleep(ms);
-        } catch (InterruptedException e) { // NOSONAR squid:S2142
-            LOG.warn("Interrupted while sleeping for {} ms: {}", ms, e);
-        }
-    }
-
-    /**
-     * Sleeps for the specified number of milliseconds after the given system
-     * time in milliseconds. If that number of milliseconds has already elapsed,
-     * does nothing.
-     *
-     * @param startTime
-     *            System time in milliseconds to sleep after
-     * @param ms
-     *            How long after startTime to sleep
-     */
-    public static void sleepAfter(long startTime, long ms) {
-        long now = System.currentTimeMillis();
-        long until = startTime + ms;
-        LOG.trace("Sleeping until {}", until);
-        if (now < until) {
-            sleep(until - now);
-        }
-    }
-
-    /**
-     * Generates a Computer Identifier, which may be part of a strategy to
-     * construct a licence key. (The identifier may not be unique as in one case
-     * hashcode could be same for multiple values, and the result may differ
-     * based on whether the program is running with sudo/root permission.) The
-     * identifier string is based upon the processor serial number, vendor,
-     * processor identifier, and total processor count.
-     * 
-     * @return A string containing four hyphen-delimited fields representing the
-     *         processor; the first 3 are 32-bit hexadecimal values and the last
-     *         one is an integer value.
-     */
-    public static String getComputerIdentifier() {
-        SystemInfo systemInfo = new SystemInfo();
-        OperatingSystem operatingSystem = systemInfo.getOperatingSystem();
-        HardwareAbstractionLayer hardwareAbstractionLayer = systemInfo.getHardware();
-        CentralProcessor centralProcessor = hardwareAbstractionLayer.getProcessor();
-        ComputerSystem computerSystem = hardwareAbstractionLayer.getComputerSystem();
-
-        String vendor = operatingSystem.getManufacturer();
-        String processorSerialNumber = computerSystem.getSerialNumber();
-        String processorIdentifier = centralProcessor.getIdentifier();
-        int processors = centralProcessor.getLogicalProcessorCount();
-
-        String delimiter = "-";
-
-        return String.format("%08x", vendor.hashCode()) + delimiter
-                + String.format("%08x", processorSerialNumber.hashCode()) + delimiter
-                + String.format("%08x", processorIdentifier.hashCode()) + delimiter + processors;
-    }
+	private static final Logger LOG = LoggerFactory.getLogger(Util.class);
```

Comment:
Nearly the entire class shows as a change in the diff because of the tabs/spaces... makes it harder to review! :)